docs/operations/cas-login-limit_de.md

```diff
@@ -5,20 +5,29 @@ Anmeldeversuchen innerhalb bestimmter Zeitgrenzen temporär gesperrt werden kön
 der Benutzer auf eine entsprechende Seite umgeleitet. Hierbei wird nicht geprüft, ob ein Benutzerkonto existiert
 (d. h. auch für nicht existierende Konten wird die Sperre durchgeführt).
 
+<strong>Hinweis: CAS berechnet aus dem failure_threshold und den range_seconds eine Fehler-Rate. Anschließend werden 
+<ins>die letzten zwei</ins> fehlerhaften Anmeldungen angeschaut und geprüft, ob die Zeit unter der Fehler-Rate liegt.
+
+Beispiel: Failure_threshold = 500; Range_seconds = 10; Fehler-Rate = 500/10 = 50
+
+=> Eine Fehler-Rate von 50 bedeutet, dass gethrottled wird, wenn zwei fehlerhafte Anmeldungen innerhalb von 0.04 Sekunden
+erfolgen: 500/10s -> 50/1s -> 5/0.1s -> 2/0.04s
+</strong>
+
 Hierzu werden die folgenden Konfigurations-Parameter des CAS-Moduls genutzt:
 
-* `login.limit.maxNumber` Setzt die maximal erlaubte Anzahl an Fehlversuchen pro Benutzerkonto.
+* `limit/failure_threshold` Setzt die maximal erlaubte Anzahl an Fehlversuchen pro Benutzerkonto.
   Wird diese Anzahl innerhalb der mit den weiteren Parametern definierten zeitlichen Rahmen
-  überschritten, wird die IP-Adresse des Nutzers temporär gesperrt.
+  überschritten, wird die IP-Adresse des Nutzers temporär gesperrt. Wird intern auf eine Fehler-Rate runtergerechnet (siehe Beispiel oben).
```

Comment:
`Fehlerrate` could be written as compound word without losing any aspects. As above, I suggest to rename it like so:

```suggestion
  überschritten, wird die IP-Adresse des Nutzers temporär gesperrt. Wird intern auf eine Fehlerrate runtergerechnet (siehe Beispiel oben).
```